app/views/welcome/index.html.slim

```diff
@@ -0,0 +1,89 @@
+nav.navbar
+  .container-fluid
+    .navbar-header
+      a.navbar-toggle.collapsed aria-expanded="false" data-target="#navbar-collapse" data-toggle="collapse" href="javascript:void(0);" 
+      a.bars href="javascript:void(0);" 
+      a.navbar-brand href="http://lmgtfy.com/?q=nicher%2Br"  NICHER+R
+      = ENV['RAILS_ENV']
```

Comment:
why it's here?

Reply:
@Sergyenko asked me to do this to distinguish production and development